nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -44,14 +44,14 @@ class TaxonomiesClassifier(SignalHandler):
     name = "render_taxonomies"
 
     def _do_classification(self, site):
-        # Get list of enabled taxonomy plugins
-        taxonomies = [p.plugin_object for p in site.plugin_manager.getPluginsOfCategory('Taxonomy')]
-        taxonomies = [taxonomy for taxonomy in taxonomies if taxonomy.is_enabled()]
-        # Prepare classification and check for collisions
+        # Needed to avoid strange errors during tests
+        if site is not self.site:
```

Comment:
This is probably caused by nonsense like FakeSite vs Site. We *could* merge it like this, but I’d prefer for this to be fixed for real. In another PR, perhaps.